test/v1/configuration.go

```diff
@@ -57,26 +57,46 @@ func PatchConfig(t pkgTest.T, clients *test.Clients, svc *v1.Configuration, fopt
 	return clients.ServingClient.Configs.Patch(svc.ObjectMeta.Name, types.JSONPatchType, patchBytes, "")
 }
 
+// WaitForConfigLatestPinnedRevision enables the check for pinned revision in WaitForConfigLatestRevision.
+func WaitForConfigLatestPinnedRevision(clients *test.Clients, names test.ResourceNames) (string, error) {
+	return WaitForConfigLatestRevision(clients, names, true)
```

Comment:
```suggestion
	return WaitForConfigLatestRevision(clients, names, true /*what is this constant?*/)
```

Reply:
I thought it was clear because the description was two lines above this line. And the WaitForConfigLatestRevision is just a few lines below with more details on the parameter. But I can add it...

Reply:
yeah I know it might seem superfluous but it does help with readability and future code changes.